File: yoda/cidmv/1DT901/ws222gx_assign1/time.py
user_input = int(input("Give a number of seconds (intger): "))

seconds = user_input % 60
minutes = (user_input // 60) % 60
hours = user_input // 3600

print(f"This corresponds to: {hours} hours, {minutes} minutes and" +
      f" {seconds} seconds.")

File: yoda/y1/1dv502/assignment-1/app/src/main/java/org/example/Time.java
package org.example;

/**
 * Time class for converting time units.
 */
public class Time {

  /**
   * Converts hours, minutes, and seconds to total seconds.
   *
   * @param hours the number of hours
   * @param minutes the number of minutes
   * @param seconds the number of seconds
   * @return the total number of seconds
   */
  public int toSeconds(int hours, int minutes, int seconds) {
    seconds += (hours * 60 * 60);
    seconds += (minutes * 60);
    return seconds;
  }
}
